flows/webhook.go

```diff
@@ -15,16 +15,28 @@ import (
 	"github.com/pkg/errors"
 )
 
-// DefaultWebhookPayload is a template that matches the JSON payload sent by legacy webhooks
-var DefaultWebhookPayload = `{
-	"contact": {"uuid": "@contact.uuid", "name": @(json(contact.name)), "urn": @(json(if(default(input.urn, default(contact.urns[0], null)), text(default(input.urn, default(contact.urns[0], null))), null)))},
-	"flow": @(json(run.flow)),
-	"path": @(json(run.path)),
-	"results": @(json(results)),
-	"run": {"uuid": "@run.uuid", "created_on": "@run.created_on"},
-	"input": @(json(if(input, object("attachments", foreach(input.attachments, attachment_parts), "channel", input.channel, "created_on", input.created_on, "text", input.text, "type", input.type, "urn", if(input.urn, object("display", default(format_urn(input.urn), ""), "path", urn_parts(input.urn).path, "scheme", urn_parts(input.urn).scheme), null), "uuid", input.uuid), null))),
-	"channel": @(json(if(input, input.channel, null)))
-}`
+// LegacyWebhookPayload is a template that matches the JSON payload sent by legacy webhooks
+var LegacyWebhookPayload = `@(json(object(
+	"contact", object("uuid", contact.uuid, "name", contact.name, "urn", contact.urn),
```

Comment:
I've simplified 

```
if(
  default(input.urn, default(contact.urns[0], null)), 
  text(default(input.urn, default(contact.urns[0], null))), 
  null
)
``` 

to just `contact.urn`... I think we needed the previously more complex expression because `contact.urn` was formatted and I can't think of any realistic scenario where these would be different.